ui-icons/dist/icons.json

```diff
@@ -5,10 +5,7 @@
   "Add200.svg",
   "Add300.svg",
   "Arrow100.svg",
-  "Arrow200.svg",
-  "Arrow300.svg",
   "Arrow400.svg",
-  "Arrow500.svg",
```

Comment:
These Arrow200, -300, and -500 were marked as deprecated in the README when icons were updated in #3001 so I manually removed them after the script added them back.

Same with MoveHorizontalTableWidget.svg.

If we do discover that we need these icons at any point, they're also still in the 1.11.0 version of the a4u package, so it shouldn't be any trouble to add them back in.